language/move-prover/scripts/README.md

```diff
@@ -6,3 +6,11 @@ There are three experimental scripts in this folder.
 `install-dotnet.sh` must be executed before `install-boogie.sh`.  There is no other dependency.
 
 The commands `boogie` and `z3` should be in your path once these scripts have finished successfully.
+
+The Move Prover looks for Boogie and Z3 executables in the environment variables BOOGIE_EXE and Z3_EXE,
+respectively.  If these environment variables are not set, Boogie/Z3 are not run on the Boogie file
```

Comment:
"If these are not set...": this apply to our test driver only, not the general tool functionality. Perhaps worthy to add "... are not run in the tests of this crate ..."

Reply:
Fixed.